### PR TITLE
Upgrade stuff - part 2

### PR DESCRIPTION
**Pull request by jhinkey**
Hi Alec,

I've added one commit since the last PR. Thanks!

### CODE REVIEW

File: docs/dxp/7.2.x/en/installation-and-upgrades/upgrading-to-liferay-7-2/01-introduction-to-upgrading-to-liferay-dxp-7-2.md [hunk 1]
```diff
@@ -32,19 +28,43 @@ If your path includes upgrading to Liferay Portal 6.2, follow the [Liferay Porta
...
+
+Before starting the upgrade process, it's important to know the biggest tasks and what tasks can be done in parallel. This facilitates planning resources for tackling such tasks. 
+
+Data upgrade is the biggest task and the articles that follow center around it by explaining how to create a data upgrade test environment, prune and tune the database, and perform the data upgrade. That said, here are the other big tasks that can be done in parallel with each other and the data upgrade tasks:
+
+- [Adapting to Feature Changes](#adapting-to-feature-changes)
+- [Upgrading Custom Code and Plugins](#upgrading-custom-code-and-plugins)
+- [Upgrading Marketplace Apps](#upgrading-marketplace-apps)
+- [Preparing a New Liferay DXP Server](#preparing-a-new-liferay-dxp-server)
+- [Installing Elasticsearch](#installing-elasticsearch)
```
Comment (jhinkey) [inline docs/dxp/7.2.x/en/installation-and-upgrades/upgrading-to-liferay-7-2/01-introduction-to-upgrading-to-liferay-dxp-7-2.md]: @Alec-Shay This listing I added doesn't add much value, and it clutters the view. It's probably best to remove it and just let these small sub-sections speak for themselves.

Comment (Alec-Shay) [reply docs/dxp/7.2.x/en/installation-and-upgrades/upgrading-to-liferay-7-2/01-introduction-to-upgrading-to-liferay-dxp-7-2.md]: @jhinkey Fair enough. The first sentence of the preceding paragraph ("Data upgrade is the biggest task"...) already sort of leads into them, so maybe that can just do the job on its own with maybe a minor tweak.

**Comment by jhinkey** [comment]
Hi Alec,

Christ Liu called out this correction that needed to be made in the Post-upgrade doc.

https://issues.liferay.com/browse/LRDOCS-7490

**Comment by Alec-Shay** [comment]
Closing this since these changes were merged into master. See: https://github.com/jrhoun/liferay-learn/pull/60

**Pull request closed without merging**